interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/DaemonSetTransformerInitializer.scala

```diff
@@ -41,8 +41,8 @@ case class DaemonSetTransformerConfig(
   private[this] val netmask = InetAddress.getByName("255.255.255.0")
 
   @JsonIgnore
-  override def mk(): NameTreeTransformer = {
-    val client = mkClient(Params.empty).configured(Label("daemonsetTransformer"))
+  override def mk(params: Params): NameTreeTransformer = {
+    val client = mkClient(params).configured(param.Label("clnt"))
```

Comment:
Yeah... I don't love it either.  "clnt" is the scope that finagle uses for its ClientStatsRecevier so there is some precedent for this abbreviation.  My main motivation was to distinguish stats for clients that we explicitly create (e.g. for talking to the k8s api etc) from stats for clients that are provisioned by linkerd routers.  ie "client" has a specific meaning in the context of linkerd and I don't want to conflate that with the one-off finagle clients created by namers and transformers.

Advice appreciated.